frontend/ui/data/settings_migration.lua

```diff
@@ -45,6 +45,57 @@ function SettingsMigration:migrateSettings(config)
         config:saveSetting("copt_word_spacing", { 100, space_condensing })
     end
 
+    if config:has("style_tweaks") then
+        local tweaks = config:readSetting("style_tweaks")
+
+        if tweaks then
+            --            base hint
+            --
+            --          | n | t | f |
+            --       ---+---+---+---+
+            -- small  n | n | t | f |
+            -- hint   t | t | t | t |
+            --        f | f | t | f |
```

Comment:
What do `n`, `t` and `f` stand for ? I really can't say :/ so it needs a comment.

Some minutes later: Oh, nil, true and false ! you may as well write them fully :)

Some minutes later: what does `hint` means ? What's the last unamed line, the resut ?

Reply:
I hope it's clearer now. The two input dimensions are the tweak setting for e.g. _classname footnotes_ and _smaller classname footnotes_, the fields are the result after the migration.

I found it helpful to write down all the cases especially because the meaning differs a bit between global and per-book settings.

Reply:
Oh, "base hint" written on 2 lines :)
"hint" feels like not the best word, I would just use "note".

Reply:
I changed it to hopefully be unambiguous and also make it clear that this is per footnote type. (diff above is outdated though)

For some reason my brain keeps replacing _"tweak"_ with _"hint"_, but I think I fixed that in all comments (probably because of `-cr-hint`)...